pkgs/by-name/ro/rosa/package.nix

```
@@ -2,31 +2,32 @@
 
 buildGoModule rec {
   pname = "rosa";
-  version = "1.2.46";
+  version = "1.2.48";
 
   src = fetchFromGitHub {
     owner = "openshift";
     repo = "rosa";
     rev = "v${version}";
-    hash = "sha256-XRoHapuH0MJNrtu+Rk/yxJqeqjNIbdGYqDB84q05rdA=";
+    hash = "sha256-qJKzJrCZKhaqoRxloTUqaRsR4/X/hoMxmDQCTNccTqk=";
   };
   vendorHash = null;
 
   ldflags = [ "-s" "-w" ];
 
   __darwinAllowLocalNetworking = true;
 
-  postPatch = ''
-    # e2e tests require network access
-    rm -r tests/e2e
-  '';
-
-  preCheck = ''
-    # Workaround for cmd/list/rhRegion/cmd_test.go:39
-    #   Failed to get OCM regions: Can't retrieve shards: Get "https://api.stage.openshift.com/static/ocm-shards.json": dial tcp: lookup api.stage.openshift.com on [::1]:53: read udp [::1]:55793->[::1]:53: read: connection refused
-    substituteInPlace "cmd/list/rhRegion/cmd_test.go" \
-      --replace-fail "TestRhRegionCommand" "SkipRhRegionCommand"
-  '';
+  # skip e2e tests package
+  excludedPackages = [ "tests/e2e" ];
+
+  # skip tests that require network access
+  checkFlags =
+    let
+      skippedTests = [
+        "TestCluster"
+        "TestRhRegionCommand"
+      ];
```

```suggestion
      ]
      ++ lib.optional stdenv.hostPlatform.isDarwin "TestCache";
```
Seems there are some [permission issues](https://logs.ofborg.org/?key=nixos/nixpkgs.359440&attempt_id=62589cca-19f6-4366-86c4-3f93476a1e46) with temporary files on Darwin. Pretty sure this is fine

---

Strangely enough, this test did not fail on aarch64-darwin (with sandbox). I'll add this one too. Again thank you for your help with this one!